pages/discover/nfts.tsx

```diff
@@ -0,0 +1,167 @@
+import { FilterOption } from '@/common/components/layouts/Filters';
```

Comment:
![image](https://user-images.githubusercontent.com/10433154/172835289-64279cb8-debd-42ac-b669-94e67f1f51f6.png)
Infinite load + search bar reduces size when no results are found / loading

I guess we need a component or section to display if no search results are found?

The term I searched for really should have had some results though, but that's a separate issue :)

Reply:
> The term I searched for really should have had some results though, but that's a separate issue :)

Yeah, we'll need to be able to pass search terms to the backend

Reply:
> Infinite load + search bar reduces size when no results are found / loading

partially fixed. I wasnt able to quickly fix the infinite load thing and probably not worth the time right now